sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md

```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with User Assigned Managed Identity
+var communicationService = await communicationManagementClient.CommunicationServices.CreateAsync(
```

Comment:
it should be the communicationServiceResourceCollection Class for resource creation

Reply:
Following the original example for resource creation should take you to the right place in regards to Joel's comment @joannajiang112628